guarddog/analyzer/metadata/pypi/typosquatting.py

```diff
@@ -13,6 +19,49 @@ class PypiTyposquatDetector(TyposquatDetector):
         popular_packages (list): list of top 5000 downloaded packages from PyPI
     """
 
+    def _get_top_packages(self) -> list:
+        """
+        Gets the package information of the top 5000 most downloaded PyPI packages
+
+        Returns:
+            list: list of package data in the format:
+                [
+                    ...
+                    {
+                        download_count: ...
+                        project: <package-name>
+                    }
+                    ...
+                ]
+        """
+
+        popular_packages_url = "https://hugovk.github.io/top-pypi-packages/top-pypi-packages-30-days.min.json"
```

Comment:
not clear if we want to hardcode it in the project or download it at runtime? or fallback to the local version?

also unclear to me how this changes the current behavior (and why we need to change it, since this PR seemed to be about npm)

Reply:
this is just a move of the previous pypi behavior (I copied the method from the base class to the implementation).

Reply:
oh ok, makes sense